app/templates/base.html

```diff
@@ -8,7 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="
     {%- block metaDescription -%}
-    The Application Development Initiative seeks to nurture student creativity and technological aptitude by creating a community and resources for student developers of all talents and interests.
+    ADI is a student group at Columbia that aims to foster a community of students interested in technology.
```

Comment:
Did you get this from somewhere? I like this a lot better than what we had before lol